ckanext/who_romania/plugin.py

```diff
@@ -151,6 +152,27 @@ def after_dataset_create(self, context, data_dict):
         if data_dict.get('private'):
             who_romania_upload.add_activity(context, data_dict, "new")
 
+    def make_middleware(self, app, config):
+        @app.after_request
+        def apply_owasp(response):
+            response.headers['Strict-Transport-Security'] = "max-age=31536000; preload"
+            response.headers['X-Content-Type-Options'] = "nosniff"
+            response.headers["X-Permitted-Cross-Domain-Policies"] = "none" #not sure about this one
```

Comment:
Seems sensible to me. I presume you've checked it doesn't impact PDF viewing... we don't use an Adobe product to do the viewing but... just in case?

Reply:
`Cross-Origin` policies might. I will have to test it - i Intend to fix the tests now to make sure the system is stable and working, and then will get back to this PR

Reply:
I cannot really test it until the issue with retreiving files is fixed I believe: https://github.com/fjelltopp/fjelltopp-infrastructure/issues/263

Reply:
What's the timeline for getting this deployed for the client?

Reply:
Shall we merge and deploy to staging - you can then manually test everything there (as we would anyway). Not best practice but it unblocks you WRT to the local dev issues.